components/back-top/style/responsive.less

```diff
@@ -1,11 +1,21 @@
 @media screen and (max-width: @screen-md) {
   .@{backtop-prefix-cls} {
     right: 60px;
+    
+    &-rtl {
+     right: auto;
+     left: 60px;
+    }
```

Comment:
Replace `right` with `insetInlineEnd`? `direction` of `&-rtl` should have been `rtl`.

Reply:
And you can get rid of overriding styles here.